src/js/components/Footer/stories/Social.js

```diff
@@ -39,10 +39,11 @@ export const Social = () => (
       </Text>
       <Box flex />
     </Main>
-    <Footer background="light-4" pad="small">
+    {/* Choosing Color similar to brand color for text "Gormmet" */}
+    <Footer background="#dadada" pad="small">
       <Box align="center" direction="row" gap="xsmall">
         <GrommetIcon color="brand" size="medium" />
-        <Text alignSelf="center" color="brand" size="small">
+        <Text alignSelf="center" color="#6F3FD4" size="small">
```

Comment:
Can we leave the Text color as `brand` and change the Footer background to be `light-3` and the Main background to `light-1`?
Usually when possible we try to use colors defined in the theme in our example stories rather than using an inline color value.

Reply:
Alright I'll try doing just that.